### DISCUSSION TITLE
Is CacheDatabase used in some active more or less big Forms project?

### DISCUSSION THREAD

**Discussion by gsgou** [reactivemarbles/CacheDatabase#50] [General]
I would like to try it out as replacement to Akavache where i face some issues. Is it ready for use?

**Comment by glennawatson** [comment]
I've used it in a couple fairly large forms app. It doesnt know about folders you'll have to specify the path directly

**Reply by gsgou** [reply]
Noticed the path requirement in the tests. Is there somewhere maybe a simple Sample available? Does it require any LinkerPreserve declarations? Tks for your feedback, keeping reactiveui alive allows me to fairly enjoy programming :)

**Reply by glennawatson** [reply]
Linker potentially. I found using system.text.json 6 with source generators eliminated a lot of issues there

Samples need to get it added. Should be close to Akavache though

**Reply by gsgou** [reply]
I dont see the requirement to call in WillTerminate(iOS), OnDestroy(Android) the Shutdown()
from Akavache implementation. Correct?
With the current VS19 stable no linking issues appear on linker behavior "SDK only".
Thanks a lot @glennawatson making it easy to move from akavache to CacheDatabase.
